### PR TITLE
fix: importing CSV file in offline reate files with correct schema

### LINKED ISSUE #1100: Importing CSV file in offline may create files with incorrect schema without warning
Now importing CSV file in offline may generate parquet files whose schema is not expected, maybe all string columns.

We should raise exception if the schema does not match.

### PR DESCRIPTION
**Pull request by yabg-shuai666**
close #1100

### CODE REVIEW

**Comment by lumianph** [comment]
@tobegit3hub let's merge this PR ASAP, and then let the user in wechat know when it is fixed.

File: java/openmldb-batch/src/main/scala/com/_4paradigm/openmldb/batch/nodes/LoadDataPlan.scala [hunk 1]
```diff
@@ -117,7 +117,12 @@ object LoadDataPlan {
 
         // do deep copy
         require(inputFile != writePath, "read and write paths shouldn't be the same, it may clean data in the path")
-        val df = spark.read.options(options).format(format).load(inputFile)
+        var struct = new StructType
```
Comment (tobegit3hub) [inline java/openmldb-batch/src/main/scala/com/_4paradigm/openmldb/batch/nodes/LoadDataPlan.scala:120]: Please add an unit test for this added function.

**Review by tobegit3hub — CHANGES_REQUESTED** [review]

**Review by tobegit3hub — APPROVED** [review]
LGTM

File: java/openmldb-batch/src/main/scala/com/_4paradigm/openmldb/batch/nodes/LoadDataPlan.scala (hunk 1 above)
Comment (tobegit3hub) [reply java/openmldb-batch/src/main/scala/com/_4paradigm/openmldb/batch/nodes/LoadDataPlan.scala:120]: We will add tests later.

**Review by vagetablechicken — APPROVED** [review]

**Pull request merged**